packages/css/src/components/alert/alert.scss

```diff
@@ -5,13 +5,8 @@
 
 .ams-alert {
   background-color: var(--ams-alert-background-color);
-  box-shadow: var(--ams-alert-box-shadow);
+  border: var(--ams-alert-border);
```

Comment:
We could split this shorthand, so we would only have to set `border-color` in the variant classes